README.md

```diff
@@ -415,7 +415,6 @@ This repository contains a curated list of awesome open source libraries that wi
 
 ## Outlier and Anomaly Detection
 * [adtk](https://github.com/arundo/adtk) ![](https://img.shields.io/github/stars/arundo/adtk.svg?style=social)  - A Python toolkit for rule-based/unsupervised anomaly detection in time series.
-* [Alibi-Detect](https://github.com/SeldonIO/alibi-detect) ![](https://img.shields.io/github/stars/seldonio/alibi-detect.svg?style=social) - Algorithms for outlier and adversarial instance detection, concept drift and metrics.
```

Comment:
Good catch, let's remove from the robustness as this library is primarily focused more on anomaly / outlier

Reply:
Done